src/main/kotlin/org/jetbrains/research/libsl/nodes/helpers/ExpressionDumper.kt

```diff
@@ -87,6 +89,14 @@ object ExpressionDumper {
         return expression.value.toString()
     }
 
+    private fun dumpIntegerLiteral(expression: IntegerLiteral): String {
```

Comment:
As far as I can see aggregating these functions will require making another parent class for number literals (since we need to pass the suffix, and current parrent "Atomic" doesn't have this field and shouldn't have), while also changing the "value" field type to "Any", which is another issue stated in other part of review. 

I think readability in this case is really more comfortable.

Reply:
While working on the next issue I've found a solution to aggregate them in one function.